### ISSUE TITLE
Optional 'ticks' attribute to turn ticks on/off

### ISSUE THREAD

**Issue by erhhung** [Wildhoney/ngRangeSlider#6]
I have a not-so-wide slider that has a big range limit (0-255) with a step of 1, and so the ticks are so close together that they loose their usefulness, so I'd like to turn them off completely.

How about supporting a Boolean `ticks` attribute (false by default) that turns the ticks off (by eliminating the huge `datalist` DOM element) if the user so chooses?

**Comment by richardsun29** [comment]
Something like this could work; I'm not sure if there is a better way to check the `ticks` property (using `$eval` maybe?)

```js
$angular.module('ngRangeSlider', []).directive('rangeSlider', function() {

    return {
        ...
        template: function(tElement, tAttrs) {
          if(tAttrs.ticks == 'true') {
            return '<section><datalist id="numbers"><option ...';
          }
          else {
            return '<section><option ...';
          }
        ...
      }

**Comment by Wildhoney** [comment]
@richardsun29 great suggestion! I don't know why I didn't think of this before. Thanks! Released as `v0.2.0`.
